cli/cmd/install-cni-plugin.go

```diff
@@ -90,15 +90,14 @@ func newCmdInstallCNIPlugin() *cobra.Command {
 
 	cmd := &cobra.Command{
 		Use:   "install-cni [flags]",
-		Short: "Output Kubernetes configs to install the Linkerd CNI Plugin",
-		Long: `Output Kubernetes configs to install the Linkerd CNI Plugin."
+		Short: "Output Kubernetes configs to install the Linkerd CNI Plugin (experimental)",
```

Comment:
i don't know how much we care about this, but this makes `install-cni` the only command description that prints beyond 80 chars when you type `linkerd`:

![screen shot 2019-02-18 at 3 28 27 pm](https://user-images.githubusercontent.com/236915/52980997-df3b7100-3391-11e9-8471-d25a574efd18.png)